packages/flutter/lib/src/material/text_field.dart

```diff
@@ -168,6 +186,38 @@ class TextField extends StatefulWidget {
   /// null, the value must be greater than zero.
   final int maxLines;
 
+  /// The maximum number of characters (Unicode runes) to allow in the text field.
```

Comment:
Unicode scalar values. "runes" is just what Dart's APIs calls them for brevity, but in the docs we should be explicit.